### PR TITLE
Also skip nft tests and add change rules from string to enum

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by yabirgb** [comment]
This is the run that skips the big tests and nfts 

https://github.com/rotki/rotki/actions/runs/3392436057/jobs/5638701911

**Comment by yabirgb** [comment]
@kelsos seems that it didn't work. I rebased but still skips them https://github.com/rotki/rotki/actions/runs/3400292417/jobs/5654637065

**Comment by kelsos** [comment]
> @kelsos seems that it didn't work. I rebased but still skips them https://github.com/rotki/rotki/actions/runs/3400292417/jobs/5654637065

Ok so exporting to GITHUB_ENV still doesn't make it available to other jobs.


Just google how to pass environment variables between jobs and adjust the rotki_ci.yml accordingly. You need to make sure that `TEST_ENVIRONMENT` gets properly passed to the backend tests

File: .github/workflows/rotki_ci.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ jobs:
       backend_tasks: ${{ steps.checker.outputs.backend_tasks }}
       frontend_tasks: ${{ steps.checker.outputs.frontend_tasks }}
       documentation_tasks: ${{ steps.checker.outputs.documentation_tasks }}
+      TEST_ENVIRONMENT: ${{env.TEST_ENVIRONMENT}}
```
Comment (kelsos) [inline .github/workflows/rotki_ci.yml:27]: nit: can you make the output name lower case please for consistency with the others?

And after I am back from vacation I will also turn it into and output on the job itself.

**Review by LefterisJP — APPROVED** [review]
LGTM. Can you also write something in the dev docs about its usage? We probably need to better define the various environments and also split the tests better in different groups

**Pull request merged**